databricks/koalas/tests/test_series.py

```diff
@@ -1489,3 +1489,16 @@ def test_floordiv_nan(self):
         kser = ks.from_pandas(pser)
 
         self.assert_eq(repr(pser.floordiv(np.nan)), repr(kser.floordiv(np.nan)))
+
+    def test_shape(self):
+        kser = ks.Series(["a", "b", "c"])
+        pser = kser.to_pandas()
+
+        self.assert_eq(repr(pser.shape), repr(kser.shape))
```

Comment:
Do we need to use `repr`?

Reply:
Yeah, everything here seems a string. So looks we won't need. Can you double check @beobest2?  Let's fix the one in https://github.com/databricks/koalas/commit/2c615ad076a262b4a3efaddca4eaa62415d6edb2 too if this is to fix.